.github/workflows/python-flake8.yml

```
@@ -30,5 +30,5 @@ jobs:
     - name: Lint with flake8
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 flake8-builtins
-        flake8 --count --max-line-length=127 --statistics mslib tests
+        pip install flake8 flake8-builtin
```

```suggestion
        pip install flake8 flake8-builtins
```

The package name is missing an "s" at the end. You can also see that in the failing CI runs output: https://github.com/Open-MSS/MSS/actions/runs/7629972323/job/20784618748?pr=2163.

---

i am so sorry i missed that my bad